src/main/java/eu/darkbot/api/managers/StarManager.java

```diff
@@ -0,0 +1,32 @@
+package eu.darkbot.api.managers;
+
+import eu.darkbot.api.entities.utils.Map;
+
+import java.util.Set;
+
+public interface StarManager {
+    String[] HOME_MAPS = new String[]{"1-1", "2-1", "3-1"};
+    String[] OUTPOST_HOME_MAPS = new String[]{"1-8", "2-8", "3-8"};
```

Comment:
While final, this is mutable. For the future should probably look into immutable collections for exposed API fields